code/modules/mob/living/simple_animal/parrot.dm

```diff
@@ -681,7 +681,7 @@
 /mob/living/simple_animal/parrot/Poly
 	name = "Poly"
 	desc = "Poly the Parrot. An expert on quantum cracker theory."
-	speak = list("Poly wanna cracker!", ":e Check the singlo, you chucklefucks!",":e Wire the solars, you lazy bums!",":e WHO TOOK THE DAMN VOIDSUITS?",":e OH GOD ITS FREE CALL THE SHUTTLE")
+	speak = list("Poly wanna cracker!", ":e What do you call a sketchy italian neighbourhood? THE SPAGHETTO SQAWWW SQAAWWW!!",":e Setup the engine, you lazy bums!","Kep't you waiting huh? Kept you waiting huh?","Venus in cancer....")

```

Comment:
No, it makes it worse, by replacing jokes that have to do with the game culture with puns and mgs references. Either don't touch those or just remove the jokes, if those are an issue.